### PR TITLE
update actions with new docs

### PR DESCRIPTION
**Pull request by zalexki**

### CODE REVIEW

File: 5.6/action.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# action.yml
+name: 'Prestashop PHP 5.6 Linter'
+description: 'Greet someone and record the time'
+inputs:
+  folder-to-exclude:
+    description: 'Folter to exclude'
```
Comment (Quetzacoalt91) [inline 5.6/action.yml]: There is a typo in here

File: 5.6/action.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# action.yml
+name: 'Prestashop PHP 5.6 Linter'
+description: 'Greet someone and record the time'
```
Comment (Quetzacoalt91) [inline 5.6/action.yml]: To be updated accordingly

File: 5.6/action.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+inputs:
+  folder-to-exclude:
+    description: 'Folder to exclude'
+    required: false
+    default: '! -path "./vendor/*"'
+runs:
+  using: 'docker'
+  image: 'Dockerfile'
+  args:
+    - ${{ inputs.folder-to-exclude }}
```
Comment (Quetzacoalt91) [inline 5.6/action.yml:13]: I'm wondering, does this work if we want to ignore several folders?

Comment (zalexki) [reply 5.6/action.yml:13]: yes it works on the validator with this
`folder-to-exclude: "! -path \"./vendor/*\" ! -path \"./files/modules/generator/*\""`

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by Quetzacoalt91** [comment]
Thank you @zalexki

**Pull request merged**